src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -336,14 +336,15 @@ private function isMaxDepthReached($class, $attribute, array &$context)
     /**
      * Gets the cache key to use.
      *
-     * @param array $context
+     * @param string|null $format
+     * @param array       $context
      *
      * @return bool|string
      */
-    private function getCacheKey(array $context)
+    private function getCacheKey($format, array $context)
     {
         try {
-            return md5(serialize($context));
+            return md5(serialize($format.$context));
```

Comment:
That does not work as $context is an array, so you would always have `Array` and a PHP notice for the array to string conversion.